Package.swift

```diff
@@ -41,7 +41,7 @@ let package = Package(
         .library(name: "VisualRecognitionV3", targets: ["VisualRecognitionV3"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/watson-developer-cloud/restkit.git", from: "1.0.0")
+        .package(url: "https://github.com/watson-developer-cloud/restkit.git", from: "1.2.0")
```

Comment:
Looks like our dependency on RestKit for SPM (Linux) -- "1.2.0" -- is different from Carthage/Cocoapods -- "1.1.0".

Is that intentional?  I suppose it might be, but it seems kinda odd.

Reply:
Thanks for catching that. I forgot to re-update the version in Carthage and Cocoapods. Everything should be using 1.2.0.